SCF-Assignment-2/Question 1/ConversionClass.java

```diff
@@ -0,0 +1,39 @@
+package Hexadecimal;
+
+/*
+ * this class includes function which will convert hexadecimal number to decimal number and vice versa.
+ */
+public class ConversionClass {
+	/*
+	 * this function will convert decimal number to hexadecimal number.
+	 */
+	public String decimalToHexa(int decimalNumber) {
+		char hexaChar[] = { '0', '1', '2', '3', '4', '5', '6', '7', '8', '9',

```

Comment:
Use a class member which will have the characters for the hexadecimals. We can use it in the functions